File: python/l00.function.py
#!/usr/bin/python

# create method (function)
def addNums (*numbers):
   sum = 0                            # initialize starting sum value
   for num in numbers:                # iterate
      sum += num                      # sum up nums
   return sum                         # return the sum

# call the method (function)
result = addNums(5, 2, 4, 3, 6)

# output results
print "The result of summation is: %d." % result

File: ruby/l00.function.rb
#!/usr/bin/ruby

# create method (function)
def add (*numbers)
   sum = 0                          # initialize starting sum value
   numbers.each {|num| sum += num } # iterate and sum up nums
   sum                              # return value
end

# call the method (function)
result = add 5, 2, 4, 3, 6

# output results
puts "The result of summation is: #{result}"